modules/ROOT/partials/individually-licensed-components/export-to-pdf/export-to-pdf-logs.adoc

```diff
@@ -0,0 +1,211 @@
+[[logs]]
+== Logs
+
+The logs from {pluginname} On-Premises are written to `stdout` and `stderr`. Most of them are formatted in JSON. They can be used for monitoring or debugging purposes. In production environments, It is recommend storing the logs to files or using a distributed logging system (like ELK or CloudWatch).
+
+=== Monitoring {pluginname} with logs
+
+To get more insight into how the {pluginname} On-Premises is performing, logs can be used for monitoring. To enable these, add the `ENABLE_METRIC_LOGS=true` environment variable.
+
+=== Log structure
+
+The log structure contains the following information:
+
+* `handler`: A unified identifier of action. Use this field to identify calls.
+* `traceId`: A unique RPC call ID.
+* `tags`: A semicolon-separated list of tags. Use this field to filter metrics logs.
+* `data`: An object containing additional information. It might vary between different transports.
+* `data.duration`: The request duration in milliseconds.
+* `data.transport`: The type of the request transport. It could be http or ws (websocket).
+* `data.status`: The request status. It can be equal to success, fail, warning.
+* `data.statusCode`: The response status in HTTP status code standard.
+
+Additionally, for the HTTP transport, the following information is included:
+
+* `data.url`: The URL path.
+* `data.method`: The request method.
+
+In case of an error, `data.status` will be equal to failed and `data.message` will contain the error message.
+
+An example log for HTTP transport:
+
+[source]
+----
+{
+  "level": 30,
+  "time": "2021-03-09T11:15:09.154Z",
+  "msg": "Request summary",
+  "handler": "convertHtmlToPdf",
+  "traceId": "85f13d92-57df-4b3b-98bb-0ca41a5ae601",
+  "data": {
+    "duration": 2470,
+    "transport": "http",
+    "statusCode": 200,
+    "status": "success",
+    "url": "/v1/convert",
+    "method": "POST"
+  },
+  "tags": "metrics"
+}
+----
+// verify if this is something we will add.
+////
+See example charts to check how to use logs for monitoring purposes.
+////
+
+=== Docker
+
+The docker has built-in logging mechanisms that capture logs from the output of the containers. The default logging driver writes the logs to files.
+
+When using this driver, use the `docker logs` command to show logs from a container. The `-f` flag can be added to view logs in real time. Refer to the link:https://docs.docker.com/engine/reference/commandline/logs/[official Docker documentation^] for more information about the logs command.
+
+> When a container is running for a long period of time, the logs can take up a lot of space. To avoid this problem, make sure that the log rotation is enabled. This can be set with the `max-size` option.
```

Comment:
Convert to note-box

Reply:
```suggestion
[NOTE]
When a container is running for a long period of time, the logs can take up a lot of space. To avoid this problem, make sure that the log rotation is enabled. This can be set with the `max-size` option.
```